chapters/tutorials/tutorial_01.rst

```diff
@@ -0,0 +1,145 @@
+==================================
+Tutorial 1
+==================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+What is Jina
+----------------------------------
+
+Tutorial
+----------------------------------
+
+At the end of this tutorial, you will have your search engine using text. You will use text as an input and get a matching text for that.
+For this example, we will use the `Kaggle Wikipedia Corpus <https://www.kaggle.com/mikeortman/wikipedia-sentences>`_
+You will understand how every part of this example works and how you could create new apps with different datasets on your own.
+
+Set-up & overview
```

Comment:
Do we have requirements before this? 

1. Linux or MaxOS
2. Python 3.7 or 3.8 installed and working 
3. A desire to summon the undeath 
etc

Reply:
I like especially point 3, but still not sure on this one cause I'm still seeing which dataset we'll use